content/blog/2019-09-23-grpc-on-dotnetcore.md

```diff
@@ -0,0 +1,61 @@
+- [**grpc-dotnet**](https://github.com/grpc/grpc-dotnet): The new implementation written entirely in C# with no native dependencies and based on the newly released .NET Core 3.0.
+
+The implementations coexist side-by-side and each has its own advantages in terms of available features, integrations, supported platforms, maturity level and performance. Both implementations share the same API for invoking and handling RPCs, thus limiting the lock-in and enabling users to choose the implementation that satisfies their needs the best.
+
+##  What's new?
+
+Unlike the existing C-Core based implementation ([Grpc.Core](https://github.com/grpc/grpc/tree/master/src/csharp)), the new libraries ([grpc-dotnet](https://github.com/grpc/grpc-dotnet)) make use of the existing networking primitives in the .NET Core Base Class Libraries (BCL). The diagram below highlights the difference between the existing **Grpc.Core** library and the new **grpc-dotnet** libraries.
+
+<p><img src="/img/grpc-dotnet.svg" alt="gRPC .NET Stack" style="max-width: 800px" /></p>
+
+On the server side, the `Grpc.AspNetCore.Server` package integrates into ASP.NET Core, allowing developers to benefit from ecosystem of common cross-cutting concerns of logging, configuration, dependency injection, authentication, authorization etc. which have already been solved by ASP.NET Core. Popular libraries in the ASP.NET ecosystem such as [Entity Framework Core (ORM)](https://github.com/aspnet/EntityFrameworkCore), [Serilog (Logging library)](https://github.com/serilog/serilog), and [Identity Server](https://github.com/IdentityServer/IdentityServer4) among others now work seamlessly with gRPC.
+
+On the client side, the `Grpc.Net.Client` package builds upon the familiar `HttpClient` API that ships as part of .NET Core. As with the server, gRPC clients greatly benefit from the ecosystem of packages that build upon `HttpClient`. It is now possible to use existing packages such as [**Polly**(Resilience and fault-handling library)](https://github.com/App-vNext/Polly) and [HttpClientFactory(manage HTTPClient lifetimes)](https://docs.microsoft.com/aspnet/core/fundamentals/http-requests) with gRPC clients.
+
+The diagram below captures the exhaustive list of all new .NET packages for gRPC and their relationship with the existing packages.
+
+<p><img src="/img/grpc-dotnet-packages.svg" alt="grpc-dotnet packages" style="max-width: 800px" /></p>
+
+In addition to the new published packages that ship as part of **grpc-dotnet**, we've also made improvements that benefit both stacks. There have been improvements to the existing packages including a more cohesive experience when working in Visual Studio especially around design-time build and language grammar support for protocol buffers.
+
+<p><img src="/img/grpc-visualstudio.png" alt="gRPC in Visual Studio 2019" /></p>
```

Comment:
optional: Picture is good, but perhaps needs a bit more context. I can see that the grammar is being highlighted,
but from just the picture I don't see any more features.
Perhaps add a picture description "gRPC project in Visual Studio 2019, beyond the .proto being syntax highlighted, the generated client/server code is immediately updated as soon .proto file is saved in the IDE and the changes are picked up by IntelliSense code completion..."